cmd/snap/cmd_first_boot.go

```diff
@@ -41,5 +39,9 @@ func (x *cmdInternalFirstBoot) Execute(args []string) error {
 		return ErrExtraArgs
 	}
 
-	return boot.FirstBoot()
+	println(`
+This command does nothing, its only there because we can not update files 
+in the writable space after they got copied out of the snap.
```

Comment:
Thanks, done. The old message was really more meant for you/the other reviewers as a reminder that we need to think a bit about the writable space :)